opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/aggregate.py

```diff
@@ -42,21 +43,25 @@ def __init__(self, config=None):
     @abc.abstractmethod
     def update(self, value):
         """Updates the current with the new value."""
+        if self.checkpointed:
```

Comment:
Nit: Is there a way to do this without introducing a new variable?

Reply:
I tried to do so by setting `self.initial_checkpoint_timestamp` to `0` in `take_checkpoint` and checking for its value being `0` in `update`. That would have saved us from another variable but the problem with that approach is that `self.initial_checkpoint_timestamp` would be `0` when the export happens, so we need to keep the value of that timestamp separate from the "flag" that we use to detect a checkpoint being taken.